projects/Mallard/src/components/update-ip-address.tsx

```diff
@@ -0,0 +1,15 @@
+import { useNetInfo } from '@react-native-community/netinfo'
+import { useEffect } from 'react'
+import { fetchAndStoreIpAddress } from '../helpers/fetch'
+
+const UpdateIpAddress = () => {
```

Comment:
@jimhunty How is this now? I've created a component for it that uses the hook as suggested.

Reply:
👍